gradle/root/release.gradle

```diff
@@ -23,23 +23,22 @@ ext {
     git_genericEmail = "<mockito.release.tools@gmail.com>"
     git_releasableBranchRegex = "master|release/.+"  // matches 'master', 'release/2.x', 'release/3.x', etc.
 
-    bintray_repo = 'maven'
-    bintray_notableRepo = 'maven-notable'
-
     pom_developers = ['szczepiq:Szczepan Faber', 'bric3:Brice Dutheil', 'raphw:Rafael Winterhalter', 'TimvdLippe:Tim van der Lippe']
     pom_contributors = []
 }
 
 allprojects {
-    plugins.withId("org.mockito.release-tools.bintray") {
+    plugins.withId("org.mockito.mockito-release-tools.bintray") {
         bintray {
             pkg {
+                repo = 'maven'
                 user = 'szczepiq'
```

Comment:
Maybe it would be good to have a separated Bintray user similar to `continuous-delivery-drone`?